### PR TITLE
Build against cosmos-sdk 0.38.0

### PR DESCRIPTION
**Pull request by alessio**
______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: cmd/gaiad/genaccounts.go [hunk 1]
```diff
@@ -51,7 +52,7 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 			if err != nil {
 				// attempt to lookup address from Keybase if no address was provided
 				kb, err := keys.NewKeyring(
-					sdk.GetConfig().GetKeyringServiceName(),
+					version.Name,
```
Comment (alexanderbez) [inline cmd/gaiad/genaccounts.go]: Why do this when the default is already this (per your PR).

File: go.mod [hunk 2]
```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	github.com/btcsuite/btcd v0.0.0-20190807005414-4063feeff79a // indirect
-	github.com/cosmos/cosmos-sdk v0.34.4-0.20200122195256-f18005d2f18b
+	github.com/cosmos/cosmos-sdk v0.38.0
```
Comment (alexanderbez) [inline go.mod]: Pin to latest master commit.

**Review by alexanderbez — CHANGES_REQUESTED** [review]
CI is failing as well.

File: cmd/gaiad/genaccounts.go (hunk 1 above)
Comment (alessio) [reply cmd/gaiad/genaccounts.go]: Fixed

**Review by alexanderbez — APPROVED** [review]
ACK 🌮

**Pull request merged**